ci/e2e-tests/suite-common.sh

```diff
@@ -23,3 +23,8 @@ echo "suite_id: $suite_id" >&2
 # Common name for all suite-level Azure resources
 suite_common_resource_name="$(printf '%s' "$suite_id" | tr -C 'a-z0-9' '-')"
 echo "suite_common_resource_name: $suite_common_resource_name" >&2
+
+# Variables related to the DPS custom allocation policy
+dps_allocation_function_name="DpsCustomAllocation"
```

Comment:
Use `'` for strings without interpolation.

Reply:
Done.